apps/condo/domains/user/UserExternalIdentityMiddleware.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express')
+const { expressErrorHandler } = require('@condo/domains/common/utils/expressErrorHandler')
+
+const { SbbolRoutes } = require('@condo/domains/organization/integrations/sbbol/routes')
+const { UserExternalIdentityRoute } = require('@condo/domains/user/routes/UserExternalIdentityRoute')
+
+const {
+    USER_EXTERNAL_IDENTITY_AUTH_PATH,
```

Comment:
These consts are not reusable. It's better to hardcode it or move it inside the Middleware file.

Reply:
hardcoded